corehq/apps/dashboard/static/dashboard/ng_partials/apps_paginated_tile.html

```diff
@@ -0,0 +1,44 @@
+<div class="col-md-3">
+    <div class="panel panel-dashboard panel-dashboard-medium">
+        <div class="panel-heading">
+            <a href="{{ default.url }}"
+               data-toggle="popover"
+               popover-title="{{ title }}"
+               popover="{{ helpText }}"
+               popover-trigger="mouseenter"
+               popover-placement="bottom"
+               title="">{{ title }}</a>
+        </div>
+        <div class="panel-body">
+            <i class="dashboard-icon dashboard-icon-medium dashboard-icon-bg {{ default.icon }}"
+               ng-show="showItemList()"></i>
+            <i class="fa fa-spinner fa-spin fa-5x" ng-show="showSpinner()"></i>
+            <div class="list-group" ng-show="showItemList()">
+                <div class="list-group-item" data-ng-repeat="item in paginatedItems" >
+                   <a href="{{ item.url }}">{{ item.name }}</a>
+                   <a class="icon" href="{{ item.release_manager_url }}"><i class="icon-alternate-download fa fa-download"></i></a>
```

Comment:
icon has been renamed  in a6e37e0
Reference to glyphicons has been deleted in 1673fca